src/main/scala/com/gu/certw/services/CertificateService.scala

```diff
@@ -0,0 +1,73 @@
+package com.gu.certw.services
+
+import java.net.URI
+import java.time.ZonedDateTime
+
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailService
+import com.amazonaws.services.simpleemail.model._
+import com.gu.certw.Logging
+import com.gu.certw.models._
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{ JsPath, Json, Reads }
+
+class CertificateService(httpClient: HttpClient, ses: AmazonSimpleEmailService) extends Logging {
```

Comment:
`httpClient` and `ses` are both each used in only one location. Would it be simpler to have `CertificateService` be an `object` and provide these as arguments?

I'm not saying one way or the other, up to you, although this is reinforced by the `null`s in the tests.